packages/card/tests/ButtonCard.test.tsx

```diff
@@ -33,34 +32,4 @@ describe("ButtonCard", () => {
 
     expect(asFragment()).toMatchSnapshot();
   });
-
-  const accessibilityProps = [
-    "tabIndex",
-    "role",
-    "aria-disabled",
-    "aria-pressed"
-  ];
-
-  it("passes accessibility props if isInput is false", () => {
-    const component = create(<ButtonCard>default</ButtonCard>);
-    const buttonCardNodeProps = component.root.findByProps({
-      "data-cy": "buttonCard"
-    }).props;
-    const propKeys = Object.keys(buttonCardNodeProps);
-
-    accessibilityProps.forEach(prop => {
-      expect(propKeys.includes(prop)).toBe(true);
-    });
-  });
-  it("does not pass accessibility props if isInput is true", () => {
-    const component = create(<ButtonCard isInput>default</ButtonCard>);
-    const buttonCardNodeProps = component.root.findByProps({
-      "data-cy": "buttonCard"
-    }).props;
-    const propKeys = Object.keys(buttonCardNodeProps);
-
-    accessibilityProps.forEach(prop => {
-      expect(propKeys.includes(prop)).toBe(false);
-    });
-  });
```

Comment:
These two tests were really problematic. I am opting to remove because of the other methods for accessibility testing we have in the application such as the storybook addon and the a11y regression tests.  Testing should be focused on behaviors.

Reply:
This makes sense as we shift to testing for behaviors.